### PR TITLE
Switch to Node.js 10

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: azuredeploy.json [hunk 1]
```diff
@@ -99,8 +99,8 @@
             "SLACK_SUBDOMAIN": "[parameters('slackTeamId')]",
             "SLACK_API_TOKEN": "[parameters('slackApiToken')]",
             "SLACKIN_RELEASE": "[parameters('slackinRelease')]",
-            "WEBSITE_NPM_DEFAULT_VERSION": "5.4.2",
-            "WEBSITE_NODE_DEFAULT_VERSION": "8.7.0",
+            "WEBSITE_NPM_DEFAULT_VERSION": "6.11.3",
```
Comment (XhmikosR) [inline azuredeploy.json]: I'm not sure if we can use just the major versions here. I think it would be better if we did.

Comment (emedvedev) [reply azuredeploy.json]: Never used Azure, but from a quick look at the docs it doesn't look like you can. And you can't require just any version either, looks like there's a selection, and you can't look at it unless you have an actual app created. Ugh.

> To find current nodejs versions available in Azure Windows WebApps, navigate to this link:
> https://<yourwebappname>.scm.azurewebsites.net/api/diagnostics/runtime

Comment (XhmikosR) [reply azuredeploy.json]: Should I just leave it as it was then? I don't use Azure so I don't even know if this works...

Comment (emedvedev) [reply azuredeploy.json]: Looks like node 10.15.2 and npm 6.4.1 should work. At least that was the recommendation in one of the (slightly outdated) articles I've found about deploying node apps on Azure.

Comment (emedvedev) [reply azuredeploy.json]: I'll just create an Azure account and check what's available later today, but this one can still be merged in the meantime :)

**Pull request merged**